blaze-server/src/main/scala/org/http4s/server/blaze/BlazeBuilder.scala

```diff
@@ -278,7 +278,11 @@ class BlazeBuilder[F[_]](
         s"BlazeServer($address)"
     }
 
-    banner.foreach(logger.info(_))
+    Option(banner)
```

Comment:
couldn't you just do this with a single `foldLeft` + an `isEmpty`?

Reply:
I may though don't see much benefits in that. This is bit more verbose but looks more idiomatic (for my taste)

Reply:
more of a code style thing, but if Ross doesn't mind I'll live with it.

Reply:
If we start doing this as a single log statement, I don't think it needs to continue to be a `Seq`.  I think this works for 0.18 for bincompat, and we might simplify it on the server builders for 0.19.